### PR TITLE
feat: added cards component

### PR DESCRIPTION
**Pull request by TejasShekar**
Added :
1. Simple Card
2. Card with badge
3. Card with dismiss
4. Card with text overlay
5. Text only card
6. Horizontal card
7. Card with shadow

Live preview : https://tejasshekar.github.io/Prime8UI/Components/Cards/cards.html

### CODE REVIEW

**Review by sonishreyas — APPROVED** [review]
Looks good to merge

File: Components/Cards/cards.html [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/a11y-dark.min.css"
+    />
+    <script src="../../Styles/docs.js"></script>
```
Comment (xii-vi) [inline Components/Cards/cards.html:24]: put script tag at the bottom of page

File: Styles/docs.css [hunk 2]
```diff
@@ -25,10 +25,18 @@
   display: flex;
 }
 
+a {
```
Comment (MaruthiPothuganti) [inline Styles/docs.css:28]: Avoid Styling with tag names.

File: Components/Cards/cards.html [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+      </nav>
+    </header>
+    <div class="wrapper">
+      <aside>
+        <ul>
+          <li><a href="../Alerts/alerts.html">Alert</a></li>
+          <li><a href="../Avatar/avatar.html">Avatar</a></li>
+          <li><a href="../Badge/badge.html">Badge</a></li>
+          <li><a href="../Buttons/button.html">Button</a></li>
+          <li><a class="active" href="#">Cards</a></li>
```
Comment (xii-vi) [inline Components/Cards/cards.html:46]: add the link to the card

File: Components/Cards/cards.html [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+        <ul class="nav-list">
+          <li><a href="#">Home</a></li>
+          <li><a class="active" href="#">Documentation</a></li>
+          <li>
+            <a href="#">Contribute <i class="fab fa-github"></i></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="wrapper">
```
Comment (xii-vi) [inline Components/Cards/cards.html:39]: give class a generic name

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Everything else looks good.

**Review by xii-vi — APPROVED** [review]
Try to use a common class for flex and all.
Few comments are there, work on that.
Rest is okay to merge.

**Pull request merged**